db.py

```diff
@@ -9,7 +9,7 @@
 config = ConfigParser()
 config.read(os.path.join(CURRENT_DIR, 'config.ini'))
 
-DATABASE = config["Database"]["File"]
+DATABASE = config.get("Database","File")
```

Comment:
Is there a reason we're preferring get? I think it'd only make sense if we already have some default value specified, which we don't. So this, won't err even if no db file is defined in the config (but It should right?)